### ISSUE TITLE
Disappearing thumbnails

### ISSUE THREAD

**Issue by coryrockliff** [krksgbr/glyphcollector#12]
When a project is closed, then reopened, some or all of the thumbnail images disappear. View > Reload / Force Reload doesn't resolve the issue. Screenshots attached.

![Screenshot 2020-04-30 09 32 51](https://user-images.githubusercontent.com/224138/80716560-c82ffd00-8ac5-11ea-8839-6179f0c59c67.png)
![Screenshot 2020-04-30 09 32 42](https://user-images.githubusercontent.com/224138/80716578-cd8d4780-8ac5-11ea-8049-47ae4a6ab0d1.png)

**Comment by krksgbr** [comment]
Thanks for the report! I have not experienced this before. I will have a look at it.
If you re-open the app, do the thumbnails load properly again?

**Comment by coryrockliff** [comment]
Nope. Thanks for looking into it! Let me know if you need any further info.
